### PR TITLE
Extraction of text messages(sms)

### LINKED ISSUE #174: Extraction of text messages
**Is your feature request related to a problem? Please describe.**
Extraction of text messages(SMS).
Implementation from the command line so that we can directly extract SMS.
Creating API which will give the extracted message data(this will help in frontend).

### PR DESCRIPTION
**Pull request by swapnalshahil**
# Description
Extraction of text message(SMS).
Implementation from the command line so that we can directly extract SMS.
Created API which will give the extracted message data(this will help in frontend).
API : /data/<case_name>/sms

Fixes #174

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

POSTMAN and CMD

CMD

2021-06-09 (2)_LI
sms.tsv file(above)

Extraction using POSTMAN(above)

API (above)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: flask-backend/requirements.txt [hunk 1]
```diff
@@ -1,23 +1,228 @@
+alabaster==0.7.12
```
Comment (shivanshuraj1333) [inline flask-backend/requirements.txt]: They're a lot of dependencies, not all are required just to introduce msg extraction.
Remove redundant imports.

File: scripts/dbm.py [hunk 2]
```diff
@@ -29,7 +30,12 @@
     '/data/system/gesture.key',
     '/data/system/cm_gesture.key',
     '/data/system/locksettings.db',
-    '/data/system/password.key'
+    '/data/system/password.key',
+    '/data/data/com.android.contacts/databases/contacts.db',
+    '/data/data/com.samsung.advp.imssettings/databases/ims.db',
```
Comment (shivanshuraj1333) [inline scripts/dbm.py]: Is this specific to Samsung only?
use generalised location for this which can run on every android device

File: React-frontend/src/components/core/HomeLogo.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ function Hl() {
                 <br />
                 <p className="h5 text-center mb-4">Open Source forensic tool for Android smartphones</p>
                 <div className="text-center">
-                <button color="elegant" href="https://github.com/scorelab/OpenMF//wiki">Know More</button>
+                <button color="elegant" href="https://github.com/scorelab/OpenMF/">Know More</button>
```
Comment (shivanshuraj1333) [inline React-frontend/src/components/core/HomeLogo.js]: Revert this change, wiki will be updated after the completion of project.

**Comment by shivanshuraj1333** [comment]
@sachinsom93 keep track of the new changes in APIs

**Review by shivanshuraj1333 — CHANGES_REQUESTED** [review]
take a look on the provided suggestions

File: React-frontend/src/components/core/HomeLogo.js (hunk 3 above)
Comment (swapnalshahil) [reply React-frontend/src/components/core/HomeLogo.js]: Sure sir!

File: scripts/dbm.py (hunk 2 above)
Comment (swapnalshahil) [reply scripts/dbm.py]: Yes, I was testing these yesterday.
Now I committed and removed those dbs locations that were specific for Samsung.

File: flask-backend/requirements.txt (hunk 1 above)
Comment (swapnalshahil) [reply flask-backend/requirements.txt]: @shivanshu1333 updated!

**Comment by swapnalshahil** [comment]
@shivanshu1333 sir, I made changes according to your suggestions.

File: flask-backend/requirements.txt (hunk 1 above)
Comment (shivanshuraj1333) [reply flask-backend/requirements.txt]: It looks like you reverted all the changes, are you sure no additional dependency is required here?

Comment (swapnalshahil) [reply flask-backend/requirements.txt]: Yes sir! No other dependency needed now. These dependencies came from the last PR #173  when setting up the flask. @sachinsom93 added the required dependencies in #176

Comment (sachinsom93) [reply flask-backend/requirements.txt]: Agreed with @swapnalshahil, sir @shivanshu1333 , actually I forgot to update requirements.txt file in #173.But, I have updated requirements.txt file in #176 .

Comment (shivanshuraj1333) [reply flask-backend/requirements.txt]: Alright!

**Review by shivanshuraj1333 — DISMISSED** [review]

**Comment by swapnalshahil** [comment]
@shivanshu1333 sir Solved the merge conflict. Please review it again!

**Review by sachinsom93 — APPROVED** [review]
